zone/client_packet.cpp

```diff
@@ -4321,6 +4336,12 @@ void Client::Handle_OP_CancelTrade(const EQApplicationPacket *app)
 		FinishTrade(this);
 		trade->Reset();
 	}
+
+	if (ClientVersion() == EQ::versions::ClientVersion::RoF2 && RuleB (Parcel, EnableParcelMerchants)) {
```

Comment:
`RuleB(Parcel, EnableParcelMerchants)`

Reply:
@Kinglykrab I left this in as I figure the client check is better than a packet and db call that would occur.  I suppose, if everyone is on RoF2, it would be more expensive, though with Ti/UF player bases, it will be less expensive.  Let me know your thoughts.

Reply:
Misunderstoond.  Updated the space item.